src/components/ToastTextToCad.tsx

```diff
@@ -411,3 +415,68 @@ function traverseSceneToStyleObjects({
     }
   })
 }
+
+export function ToastPromptToEditCadSuccess({
+  toastId,
+  token,
+  data,
+  oldCode,
+}: {
+  toastId: string
+  oldCode: string
+  data: TextToCadIteration_type
+  token?: string
+}) {
+  const modelId = data.id
+
+  return (
+    <div className="flex gap-4 min-w-80">
+      <div className="flex flex-col justify-between gap-6">
+        <section>
+          <h2>Prompt to edit successful</h2>
+          <p className="text-sm text-chalkboard-70 dark:text-chalkboard-30">
+            Prompt: "
+            {data?.prompt && data?.prompt?.length > PROMPT_TRUNCATE_LENGTH
+              ? data.prompt.slice(0, PROMPT_TRUNCATE_LENGTH) + '...'
+              : data.prompt}
+            "
+          </p>
+          <p>Do you want to keep the change?</p>
+        </section>
+        <div className="flex justify-between gap-8">
+          <ActionButton
+            Element="button"
+            iconStart={{
+              icon: 'close',
+            }}
+            data-negative-button={'reject'}
+            name={'Reject'}
+            onClick={() => {
+              sendTelemetry(modelId, 'rejected', token).catch(reportRejection)
```

Comment:
Oh good I was gonna ask if we were including telemetry!

Reply:
This component is just a copy paste of yours, with heaps cut out of it, so much simpler without the threejs scene and what not, there's also no copy code for the web app since it's already updating the editor code, so I kept these bits. But now that I think of it. I think this is an oversite because this sendTelemtry is going to the wrong URL, I'll ask about it.

Reply:
I might remove this for now with a TODO this is sending to 
`VITE_KC_API_BASE_URL + '/user/text-to-cad/' + id + '?feedback=' + feedback`

Thanks for prompting me on this.